packages/utilities/src/focus.ts

```diff
@@ -35,11 +36,28 @@ export function getLastFocusable(
   currentElement: HTMLElement,
   includeElementsInFocusZones?: boolean): HTMLElement | null {
 
-  return getPreviousElement(rootElement, currentElement, true, false, true, includeElementsInFocusZones);
+  return getPreviousElement(rootElement, currentElement, true /*checkNode*/, false /*suppressParentTraversal*/,
+    true /*traverseChildren*/, includeElementsInFocusZones);
+}
+
+/**
+ * Gets the first tabbable element.
+ * The difference between focusable and tabbable is that tabbable elements are focusable elements that also have tabIndex != -1.
+ *
```

Comment:
I would add `@param` docs espousing a little bit on what root/current refers to.

Reply:
done.